cmd/release-controller/sync_verify.go

```diff
@@ -13,6 +14,7 @@ import (
 
 func (c *Controller) ensureVerificationJobs(release *Release, releaseTag *imagev1.TagReference) (VerificationStatusMap, error) {
 	var verifyStatus VerificationStatusMap
+	retryQueueDelay := -1 * time.Second
```

Comment:
Don't use negative numbers for this, make this `var retryQueueDelay time.Duration` and you should only delay on values > 0.